internal/data/trace.go

```diff
@@ -135,10 +139,18 @@ func (m *Span) TraceID() TraceID {
 	return TraceIDFromBytes(m.orig.TraceId)
 }
 
+func (m *Span) TraceIDBytes() []byte {
+	return m.orig.TraceId
+}
+
```

Comment:
Why do we need this? What prevents us from using `TraceID()`? We will just need to add a method to get []byte from TraceID type. I think it will be nicer and will allow to keep `Span`'s exported interface minimal.

(Side note: the reason for TraceID existence is to have strong-typing to avoid confusing different ids and e.g. assigning traceid to spanid).

Reply:
I wasn't sure about this. Yes, it's not that clean interface but helps to avoid immediate  wrapping->unwrapping for the usecase when only byte array is needed.

Reply:
Can you please check if it actually does wrapping/unwrapping in the generated code or it is noop and is inlined? If it is not a noop I would get rid of it and use `[]byte` directly?

Reply:
Checked. wrapping/unwrapping appeared to be noop indeed. I changed the code accordingly

Reply:
Good, I was worried that Go compiler will not optimize it out :-)